.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build
```

Comment:
Name should be more specific, "Build Python Package" or something (same applies to file name).